web/app/js/components/BreadcrumbHeader.jsx

```diff
@@ -14,6 +14,7 @@ const routeToCrumbTitle = {
   top: 'Top',
   routes: 'Top Routes',
   community: 'Community',
+  gateways: 'Gateways',
```

Comment:
This is a bonus fix unrelated to translations - I noticed that the header title when viewing `/gateways` was not properly capitalized